samwise.js

```diff
@@ -1,17 +1,18 @@
-const filteredArray = array.filter(item => item > 10);
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!